### PR TITLE
Milestone-6/Blog app - forms

### PR DESCRIPTION
**Pull request by edwardoluyara**
Here in this milestone I have been able to implement the following:
- Create forms to perform the following functions:
  - Create a Post on behalf of the current_user.
  - Create a comment on behalf of the current_user.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
-  has_many :posts, foreign_key: 'author_id'
-  has_many :comments, foreign_key: 'author_id'
-  has_many :likes, foreign_key: 'author_id'
+  has_many :posts, class_name: 'Post', foreign_key: 'author_id'
+  has_many :comments, class_name: 'Comment', foreign_key: 'post_id'
+  has_many :likes, class_name: 'Like', foreign_key: 'post_id'
```
Comment (Gambit142) [inline app/models/user.rb]: - [ ] Kindly change the `foreign_key:` `author_id`. Remember that the `User` is connected to the tables through `author_id`. When I try to create `comments` and `likes,` I get that error. Kindly fix it. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @oluyaratosin123 👋, 

Good job so far!

## Highlights

- Implementation for creation of `post` works well ✔️
- Implementation for creation of `comments` works well ✔️
- Implementation for creation of `likes` works well ✔️ 

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_indly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: app/models/user.rb (hunk 1 above)
Comment (edwardoluyara) [reply app/models/user.rb]: Hello @Gambit142 so do you mean I should change the foreign_key `post_id` to `author_id`?

Comment (Gambit142) [reply app/models/user.rb]: Hello @oluyaratosin123, yes, kindly change it from `post_id` to `author_id`. 👍

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @oluyaratosin123 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**